Tests/UIKitTests/UILayoutPriorityExtensionsTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UILayoutPriorityExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by diamantidis on 8/19/18.
+//  Copyright © 2018 SwifterSwift. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+
+import XCTest
+@testable import SwifterSwift
+
+final class UILayoutPriorityExtensionsTests: XCTestCase {
+
+    func testFloatLiteralInitializer() {
+
+        var priority: UILayoutPriority = 0.5
+        XCTAssertEqual(UILayoutPriority(rawValue: 0.5), priority)
+
+        priority = -0.5
+        XCTAssertEqual(UILayoutPriority(rawValue: -0.5), priority)
+
+        priority = 0.0
+        XCTAssertEqual(UILayoutPriority(rawValue: 0.0), priority)
+
+        // test against `Float.greatestFiniteMagnitude` literal
+        priority = 3.402823e+38
```

Comment:
Why didn't you use the literal value itself, instead of hard-coding it?

Reply:
It should be a literal. If you try to use `Float.greatestFiniteMagnitude`, Xcode will complain.

Reply:
Interesting, I didn't realise that `ExpressibleByFloatLiteral` only allowed for hard-coded values, not anything of a float type. In that case there isn't really a need for this test, since it's not actually the `greatestFiniteMagnitude` (you can check for yourself by doing a comparison, but they're very slightly different). Same for the negative version. You can remove them both.

Reply:
Removed